### PR TITLE
future.settings: Register runner.output.* using the new API

### PR DESCRIPTION
**Pull request by beraldoleal**

### CODE REVIEW

File: avocado/core/__init__.py [hunk 1]
```diff
@@ -69,6 +65,28 @@ def register_core_options():
...
+    future_settings.register_option(section='runner.output',
+                                    key='color',
+                                    default='auto',
+                                    help_msg=help_msg)
+
+    help_msg = 'Use UTF8 encoding (True or False)'
+    future_settings.register_option(section='runner.output',
+                                    key='utf8',
+                                    key_type=bool,
+                                    default=True,
```
Comment (willianrampazzo) [inline avocado/core/__init__.py:87]: Before the default value was `None`, now we are setting it to `True`. Does it change the current behavior? I'm fine if we agree to set it to `True` by default, I just got curious about how it is behaving now.

Comment (beraldoleal) [reply avocado/core/__init__.py:87]: I took the opportunity to change this. The value is a bool, should not have a "none" behavior. It was inconsistent with its type.

File: avocado/core/__init__.py [hunk 2]
```diff
@@ -15,16 +15,12 @@
...
 
 def register_core_options():
-    streams = (['"%s": %s' % _ for _ in BUILTIN_STREAMS.items()] +
-               ['"%s": %s' % _ for _ in BUILTIN_STREAM_SETS.items()])
-    streams = "; ".join(streams)
     help_msg = ("List of comma separated builtin logs, or logging streams "
-                "optionally followed by LEVEL (DEBUG,INFO,...). Builtin "
-                "streams are: %s. By default: 'app'" % streams)
+                "optionally followed by LEVEL (DEBUG,INFO,...). See "
+                "documentation for detailed explanation.Default: 'app'")
```
Comment (clebergnu) [inline avocado/core/__init__.py:23]: I think it's better to not regress this feature, as it's simple to work around this ordering issue (and there are bonus points for even "better refactored" code).  See https://github.com/clebergnu/avocado/commit/9d07dd0e0565735c5312c2a1cdf75e2a8d20575f

Comment (beraldoleal) [reply avocado/core/__init__.py]: Ok @clebergnu,  I just force-pushed here with your suggestion. Let me know if you need a v2 or this is ok.

**Comment by beraldoleal** [comment]
> This pull request **introduces 1 alert** when merging [4cc1af6](https://github.com/avocado-framework/avocado/commit/4cc1af6a113b532a8ff4da1e6425f874db78b2c4) into [86624c6](https://github.com/avocado-framework/avocado/commit/86624c6b877d07d2d462b2ac2203735238c2d369) - [view on LGTM.com](https://lgtm.com/projects/g/avocado-framework/avocado/rev/pr-3abfdf2769029bb42fe0a04c1788c2e5007dbc83)
> 
> **new alerts:**
> 
>     * 1 for Variable defined multiple times

This was fixed on a force-push here.

**Pull request merged**